react/react1/week2/react1-week2/src/Main.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useState } from "react";
+import TodoHeader from "./components/TodoHeader";
+import TodoList from "./components/TodoList";
+import './TodoListStyle.css'
+
+const todoArray = [
+    {
+      id: 1,
+      description: "Get out of bed",
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+    },
+  ];
+
+
+
+
+function Main(){
+    const [todos, setTodos]=useState(todoArray)
+    const buttonClick =()=>{
+        const todoArrayLen =todos.length
+        const addTodoList=[
+            {
+              id: todoArrayLen +1,
+              description: `Random text ${todoArrayLen +1}`,
+            }          
+                  ];
+        const nextTodoList=todos.concat(addTodoList)
+        setTodos(nextTodoList)
```

Comment:
Try this         setTodos([...todos, {
                       id: Date.now(),
                       task: 'review Haile's PR',
}])

Reply:
Hi Haile I tried it and it works but how can I add more tasks with different key id

Reply:
id: Date.now() is unique, every time it gives you a different value,
you can try console.log(Date.now()).  key={todo.id}, since your id:Date.now()

Reply:
Yes I get it, but whenever I click the bottom I get the same task how can I add a new task ?

Reply:
Hey, guys, as I clarified on the top, you don't need to make different todos right now.
Please be patient until I answer, I'm doing my best to review your PRs, work, and study as well. 😅